_health-care/_js/_components/error-message.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * An error message to be included conditionally by question components in an error state.
+ *
+ * @constructor
+ */
+class ErrorMessage extends React.Component {
+  render() {
+    return (
+      <span className="usa-input-error-message" id="input-error-message" role="alert">{this.props.message}</span>
```

Comment:
It's gotta come in from a prop doesn't it? I don't think we should be querying our containing component.